shell/browser/native_browser_view_mac.mm

```diff
@@ -232,10 +232,12 @@ - (BOOL)mouseDownCanMoveWindow {
 void NativeBrowserViewMac::UpdateDraggableRegions(
     const std::vector<gfx::Rect>& drag_exclude_rects) {
   auto* iwc_view = GetInspectableWebContentsView();
-  auto* web_contents = GetWebContents();
-  if (!(iwc_view && web_contents))
+  if (!iwc_view)
```

Comment:
This would work, but for readability's sake:

```cpp
if (!inspectable_web_contents_)
     return;
auto* web_contents = inspectable_web_contents_->GetWebContents();
auto* iwc_view = GetInspectableWebContentsView();
```

lets us do fewer checks and is more readable :)

Reply:
ok got it